.github/workflows/prod_upload_to_s3.yml

```diff
@@ -0,0 +1,71 @@
+name: Copy files to Production S3 bucket
+
+on:
+  push:
+    tags:
+      - v[1234].[0-0]+.[0-9]+
+
+permissions:
+  id-token: write
+  contents: write
+
+env:
+  PRODUCTION_ACCOUNT_ID: "296255494825"
+
+jobs:
+  s3-backup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+
+      - name: Generate Tailwind CSS file
+        run: |
```

Comment:
you are generating these files on the main branch, but then getting the tag after, should you checkout infra tag first?

Reply:
Yea this makes more sense, we shouldn't run the build unless we know we're going to do a push.

Reply:
I think we can move the if to the job level instead of redundantly at each step as well...